src/charts/scatter-plot.js

```diff
@@ -713,6 +717,21 @@ define(function(require) {
             return this;
         };
 
+        /**
+         * Exposes ability to set the format of x-axis values
+         * @param  {String} _x               Desired height for the chart
```

Comment:
Is this working fine with a format of ' ' as default?

Reply:
Yes! I believe ‘’ let’s it figure out the format on its own. But I tried null or undefined and it was throwing an error. If there is some other better ways you know, we can do that instead!

Reply:
Awesome, sounds good!